kobocat-template/templates/export_list.html

```diff
@@ -5,12 +5,6 @@
 {% block before-content %}
 {% load i18n %}
 
-<div class="sub-header-bar">
```

Comment:
Let me validate, but I think it's because of legacy exports. They are still accessible from the new UI.
Legacy UI is dead, long live the **new** UI :-P